src/components/dialog/alert.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import Button from 'react-toolbox/lib/button';
+import grid from '../../../node_modules/flexboxgrid/dist/flexboxgrid.css';
```

Comment:
For importing from npm modules, there's not need to use relative path. `flexboxgrid/dist/flexboxgrid.css';` would work fine.